src/applications/disability-benefits/all-claims/pages/militaryHistory.js

```diff
@@ -65,8 +76,24 @@ export const schema = {
       required: ['servicePeriods'], // required in fullSchema
       type: 'object',
       properties: {
-        servicePeriods:
-          fullSchema.properties.serviceInformation.properties.servicePeriods,
+        // fullSchema.properties.serviceInformation.properties.servicePeriods
+        servicePeriods: {
```

Comment:
@Mottie I think we can remove this now that vets-json-schema has been updated. Is that right?